components/common/src/templating/hooks.rs

```diff
@@ -82,6 +82,11 @@ pub trait Hook: fmt::Debug + Sized + Send {
         let has_template = template.exists();
         let has_deprecated_template = deprecated_template.as_ref().map_or(false, |t| t.exists());
 
+        if has_template && file_name == "reload" {
+            outputln!(preamble package_name, "The '{}' hook has been deprecated. You should use the 'reconfigure' hook instead.",
+                file_name);
+        }
+
```

Comment:
I think this is difficult to achieve in this specific case. The `Hook` trait is defined in the `common` crate. This is where we would need access to `ReloadHook` which is defined in the `sup` crate. We could refactor the code to make this work, but since we plan on removing this check anyways, I am going to propose we leave it for now.

Reply:
ah, that's right. 

Sounds good... let's leave it as-is.